infrastructure/helm/quantumserverless/README.md

```diff
@@ -77,3 +108,9 @@ For our Ray Charts dependencies we are using the configuration created by the Ra
 - For Ray Cluster they provide you with a commented initial setup in their [values.yaml](https://github.com/ray-project/kuberay-helm/blob/main/helm-chart/ray-cluster/values.yaml).
 
 - For Ray Api Server you can read their [values.yaml](https://github.com/ray-project/kuberay-helm/blob/main/helm-chart/kuberay-apiserver/values.yaml).
+
+**Keycloak**
+
+- The initial user ID and password for both keycload console(adminUser/adminPassword) and Ray dashboard(keycloakUserID/keycloakPassword) can be changed in the values.yaml file. It is good to change them before apply the helm.
+- Keycloak console can be accessed at http://LOCAL-IP:31059/.  Its initial user ID and password are "admin" and "passw0rd".
+- Ray dashboard can be accessed at http://localhost/.  Its initial user ID and password are "user" and "passw0rd".
```

Comment:
This one still doesn't work for me ...  `http://localhost/` gives me an `unable to connect` error